travis_pypi_setup.py

```diff
@@ -74,7 +75,7 @@ def prepend_line(filepath, line):
 
     lines.insert(0, line)
 
-    with open(filepath, 'w') as f:
+    with open(filepath, "w") as f:
```

Comment:
[inline-plz]: `prospector: pylint: Variable name "f" doesn't conform to snake_case naming style (invalid-name)`